packages/scene-composer/src/components/panels/scene-components/DataOverlayComponentEditor.spec.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { act, render } from '@testing-library/react';
+import wrapper from '@awsui/components-react/test-utils/dom';
+
+import { KnownComponentType } from '../../../interfaces';
+import { IDataOverlayComponentInternal, ISceneNodeInternal, useStore } from '../../../store';
+import { Component } from '../../../models/SceneModels';
+import { mockProvider } from '../../../../tests/components/panels/scene-components/MockComponents';
+
+import { DataOverlayComponentEditor } from './DataOverlayComponentEditor';
+
+jest.mock('@awsui/components-react', () => ({
+  ...jest.requireActual('@awsui/components-react'),
+}));
```

Comment:
We shouldn't be "unmocking" polaris, this is a temporary hack that allows old tests to continue functioning until they migrate off.

We shouldn't be including polaris functionality in our tests.